src/main/java/org/spongepowered/api/text/Text.java

```diff
@@ -319,23 +318,29 @@ public String toString() {
 
     /**
      * Represents a {@link Text} placeholder that can be replaced with another
-     * Text by {@link Texts#format(Text, Map)}.
+     * Text by {@link Texts#format(Text, Function)}.
      *
      * @see TextBuilder.Placeholder
      */
     public static class Placeholder extends Text {
```

Comment:
I feel uncomfortable with the notion of text placeholders as a subclass of text. Text and its subclasses were originally designed to represent Minecraft text objects, and Placeholder being a subclass of text doesn't really make sense in that context. Does that make sense?